### ISSUE TITLE
can't operate Files which doesn't have condition attribute.

### ISSUE THREAD

**Issue by YU-SUKETAKAHASHI** [adansons/base#59]
# Error messages, stack traces, or logs
we can not operate Files which doesn't have `condition` attribute.
```python
    413             files.reprtext = files.reprtext + other.reprtext
    414             files.expression += " + " + other.expression
--> 415             files.conditions = self.conditions + "," + other.conditions
    416             files.query = sorted(
    417                 set([*(self.query), *(other.query)]),

TypeError: can only concatenate str (not "NoneType") to str
```

# Steps to reproduce
I will change the initial value of `condition` : None -> ''
or 
stop concatenating conditions and query, because it is unnecessary.

# Additional context (optional)

**Comment by 31159piko-suke** [comment]
Thanks! I agree.
Files don't need to remember its conditions and query, so I will remove.
Rather, attributes of conditions and query in merged File Class are incorrect and confuse the user.

**Comment by 31159piko-suke** [comment]
solved by #60
